### ISSUE TITLE
Did you have the decent result?

### ISSUE THREAD

**Issue by cswin** [amr4i/EmotionDetection#9]
I did try this implementation on the dataset used in the paper.  But I did not get a similar result compared to the loss error mentioned in the paper.    
@ asadabbas09  you said you had similar results. can you be more specific about the results what you got?

**Comment by cswin** [comment]
[results_04282207_normalized.xlsx](https://github.com/amr4i/EmotionDetection/files/3128241/results_04282207_normalized.xlsx)
This is my result.  I doubt the paper idea is really working for emotion extraction.   @amr4i @ asadabbas09

**Comment by asadabbas09** [comment]
@cswin My best model got val_loss 1.55 for arousal and 1.88 for valence. Which is not as good as authors. I used 80% data for training and 20% for validation. 

I also wrote couple of emails to authors regarding 59-dim? colour features, maybe they can boost the results but haven't heard back.

**Comment by cswin** [comment]
Thank you Asad Abbas.
 I got vey bad results.   The validation loss error  did not decrease
during training.

I also extracted 1721 dimensional features by using VGG16 for object
features, ResNet50-dilated for segmentation, and gist and LBP for local
features.

I doubt if my bad results are due to the segmentation features, which are
extracted with original github code you shared with me rather than the code
in this GitHub.

Do you have any suggestions for me? Thank you very much.

**Comment by cswin** [comment]
@asadabbas09 In addition, did you use your trained model to predict the valence or arousal scores?  How about that results?

**Comment by asadabbas09** [comment]
@cswin I'm not sure problem might be with the extracted features. I've used the semantic code from this GitHub repo as well as from authors repo. Authors repo with updated encoder-decoder network gives a slightly better result.

Yes, I've used my own trained network for val result.

**Comment by cswin** [comment]
@asadabbas09 
if possible you can share the extracted features or the trained model with me to help me find the problem?  Thank you very much.

**Comment by cswin** [comment]
@asadabbas09 In addition,  what's your learning rate for training?

**Comment by cswin** [comment]
@asadabbas09 Thank you.  I changed the learning rate to be larger, the loss error seems to be normal and starts to decrease.     Are your predicted valence and arousal scores close to the paper?

**Comment by cswin** [comment]
I got similar results.  Thank you.

**Comment by cherrypiecoco** [comment]
Hi @cswin and @asadabbas09 In the end, did you normalize the labels or use it as is? Also what was the learning rate that you used?
